### ISSUE TITLE
Unable to locate devices. Please try running the program in Admin Mode.

### ISSUE THREAD

**Issue by Sincool** [PsiLupan/MakeLobbiesGreatAgain#55]
Hi. I'm getting this issue https://puu.sh/vZRXB/b5a71c32fd.png everytime I start MLGA. 
I tried everything. I did the .bat file, reinstall npcap, rebooted and everything, even deactivating uac. No matter what I do it will give me that error. 
Any fix / workaround?

**Comment by PsiLupan** [comment]
Install WinPcap and see if it works better for you.

**Comment by Sincool** [comment]
It still gives the same error.
I uninstalled npcap (winpcap would tell me that there's a newer version installed otherwise) -> rebooted -> installed winpcap and tried it, still the same error -> rebooted -> still same error.

**Comment by Derby8** [comment]
I just got this issue resolved. I deleted my MLGA folders, Nmap and Java. Reinstalled everything from scratch and made sure to tick the box Install Npcap in WinPcap API-compatible Mode" during installation. pretty sure AI missed that the last few times I tried fixing this issue. 

Hope this helps.

**Comment by Sincool** [comment]
I have always ticked the box and PsiLupan suggested that I'd install WinPcap instead of npcap, which I tried, and if you pay attention WinPcap is exactly what you tick when you install npcap. Neither worked. But tomorrow I will try a reinstall of Java as well, maybe that would fix it. As far as I can remember, I tried to install a developer version of java in the past and then uninstalled, it, the normal version of it remained but maybe reinstalling it will be the fix I need. 

I doubt it, though. I'll come in with an update tomorrow when I'll try.
